### ISSUE TITLE
bbc-a11y errors caused by mPulse beacon

### ISSUE THREAD

**Issue by rossgaskell** [bbc/simorgh#1635]
**Is your feature request related to a problem? Please describe.**
We are currently blocked from merging #1606 because the mPulse beacon snippet creates an iframe that has flagged issues with bbc-a11y regarding expected elements (main, h1 etc).

**Describe the solution you'd like**
- Add a hide rule to the bbc-a11y config to hide errors caused by this iframe. The Iframe should not be visible/navigable by users so the issues flagged are false negatives.

**Describe alternatives you've considered**
- Remove the beacon from the CI environment.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
Example bbc-a11y failure
![image](https://user-images.githubusercontent.com/21201639/56662158-44b22380-669b-11e9-9a88-4a93c800df43.png)

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by rossgaskell** [comment]
Closed via #1606
